gitbutler-ui/src/lib/components/BranchLanePopupMenu.svelte

```diff
@@ -121,20 +121,22 @@
 {/if}
 
 <Modal width="small" bind:this={renameRemoteModal}>
-	<TextBox label="Remote branch name" id="newRemoteName" bind:value={newRemoteName}></TextBox>
-
-	<svelte:fragment slot="controls" let:close>
-		<Button color="neutral" kind="outlined" on:click={close}>Cancel</Button>
-		<Button
-			color="primary"
-			on:click={() => {
+	<svelte:fragment let:close>
+		<form
```

Comment:
Instead of introducing a form with `on:submit`, how about having an `on:keydown` handler on the textbox? If we make a submit function we could do e.g, and we don't need to introduce restprops.

```ts
  on:keydown={(e) => {
    if (e.detail.key == 'Enter') submit();
  }}
```

Reply:
@mtsgrd What would you do if we had more than one input?

I know that this isn't the case, but I'm not sure if a form with single input should be treated as a special case.

Reply:
Also, with the `on:submit` approach, you don't have to hard-code the `Enter` key.